chrome_extension/manifest.json

```diff
@@ -55,7 +55,7 @@
     "images/icon_password_16.png",
     "images/icon_password_24.png",
     "images/logo_inverted.png",
-    "http-auth.html"
+    "ui/*"
```

Comment:
Yes, I tried to remove this line and iframes did't load. Also previously we had an issue with _http-auth.html_ when it wasn't copied when we pack extension and I added _ui_ folder to our build scripts.